node.js

```diff
@@ -10,7 +10,7 @@ module.exports = function shopifyNodePreset(_api, options = {}) {
     presets: [
       [require.resolve('@babel/preset-env'), {
         modules,
-        useBuiltIns: 'entry',
+        useBuiltIns: options.useBuiltIns || 'entry',
```

Comment:
Should we just change this to usage? I am not sure why we ended up picking entry.

Reply:
I'd be open to that 👍